src/content/releases/v0.4.0.mdx

```diff
@@ -0,0 +1,117 @@
+
+Every space now has a dedicated Docs & Files section. Upload, organizate, and
+find documents right where your team is working. No more searching through email
+threads or Slack channels to find that one document you need. Documents live
+where the work happens.
+
+<BigImage src={docsFiles} alt="Docs & Files" />
+
+## Redesigned homepage
+
+We've rebuilt the homepage for clarity. You'll see all your spaces laid out
+with the people who are part of them.
+
+On the bottom of the page, you'll find a feed of all the latest updates from
+your spaces, so you can stay up-to-date with what's happening in your company.
+
+<BigImage src={homepage} alt="New Homepage" />
+
+## New space dashboard
+
+We’ve redesigned the space dashboard to give you a better overview of what’s
+happening in your space. The new dashboard provides a snapshot of all the
+projects, goals, and discussions in the space, so you can quickly see what’s
+going on and where you need to focus your attention.
+
+<BigImage src={spaceDashboard} alt="New Space Dashboard" />
+
+## Unified Goals & Projects
+
+Goals & Projects should be two sides of the same coin. Goals are the outcomes
+you want to achieve, and Projects are the work you need to do to get there. In
+Operately v0.4, we've unified Goals & Projects into a single view, so you can
+see how they relate to each other and track progress more easily.
+
+<BigImage src={unifiedGoalsAndProjects} alt="Unified Goals & Projects" />
+
+## Company admins and owners
+
+Administering your Operately account is now easier than ever. We have split the
+role of Company Admin into two separate roles: Admins and Owner.
+
+**The Admin role** is perfect for users who need to manage the day-to-day operations
+of your Operately account. They can invite new users, adjust company settings,
+adjust profiles of employees, and offboard users who are no longer with the company.
+The role is perfect for HR, IT, Security, Compliance, and other roles that need
+to manage the account.
+
+**The Owner role** is for the person who has ultimate control over the account. They
+can do everything an admin can do, plus they can delete the account and manage admins
+and owners. Owners have full and direct access to all data in the account, including 
+private spaces, projects, and goals.
+
+<BigImage src={adminsOwners} alt="Company Admins and Owners" />
+
+## Draft messages
+
+Often you need multiple passes to get a message just right, or you need to
+collaborate with your team before sending it. In Operately v0.4, you can now
+save messages as drafts, review them with your team before sending, or come
+back to them later and finish them.
+
+<BigImage src={drafts} alt="Draft Messages" />
+
+---
+
+## Get started: Self-host in 5 minutes
+
+[Follow the self-hosted installation guide][install] to take Operately for
+a spin: you can install it on a small (1 CPU, 2GB) machine
+in 5 minutes.
+
+## Sign up for the cloud waiting list
```

Comment:
Let's replace this with a straight call for signing up, and put it above self-hosting.

Reply:
Uh, super good catch. I blindly copied the footer assuming that it stays constant, while in actually we have made major changes.

Reply:
Tweaked a lil bit.